<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>881</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow{
background-color:white;
}

#centralwidget{
background-color:white;
}


#page {
background-color:white;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QStackedWidget" name="mainStack">
      <property name="styleSheet">
       <string notr="true">#headerLine1 {
color: #10576D;
}

#headerLine2{
color:#10576D;
}

</string>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="inside_experiment"/>
      <widget class="QWidget" name="outside_experiment">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="outside_experiment_header_layout" stretch="1,1,2,1">
          <property name="spacing">
           <number>20</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>10</number>
          </property>
          <property name="bottomMargin">
           <number>5</number>
          </property>
          <item>
           <widget class="QPushButton" name="historyButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>110</width>
              <height>60</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>80</height>
             </size>
            </property>
            <property name="accessibleName">
             <string>histórico</string>
            </property>
            <property name="styleSheet">
             <string notr="true">qproperty-iconSize: 80px;</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="images/resources.qrc">
              <normaloff>:/lightHistory.png</normaloff>:/lightHistory.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>80</width>
              <height>80</height>
             </size>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="experimentButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>110</width>
              <height>60</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>80</height>
             </size>
            </property>
            <property name="accessibleName">
             <string>experimento</string>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="images/resources.qrc">
              <normaloff>:/lightExperiment.png</normaloff>:/lightExperiment.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>80</width>
              <height>80</height>
             </size>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="configurationButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>110</width>
              <height>60</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>80</height>
             </size>
            </property>
            <property name="accessibleName">
             <string>configurações</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normalon>:/lightConfiguration.png</normalon>
             </iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>100</width>
              <height>100</height>
             </size>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="headerLine">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>10</horstretch>
            <verstretch>10</verstretch>
           </sizepolicy>
          </property>
          <property name="sizeIncrement">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">#headerLine {
color: #10576D;
}</string>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="lineWidth">
           <number>4</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QStackedWidget" name="outside_experiment_stack">
          <property name="currentIndex">
           <number>2</number>
          </property>
          <widget class="QWidget" name="history_page">
           <widget class="QLabel" name="label">
            <property name="geometry">
             <rect>
              <x>60</x>
              <y>120</y>
              <width>591</width>
              <height>141</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>26</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Página do histórico em construção.</string>
            </property>
           </widget>
          </widget>
          <widget class="QWidget" name="configuration_page">
           <widget class="QLabel" name="label_2">
            <property name="geometry">
             <rect>
              <x>50</x>
              <y>130</y>
              <width>701</width>
              <height>141</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>26</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Página das configurações em construção.</string>
            </property>
           </widget>
          </widget>
          <widget class="QWidget" name="experiment_page">
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="topMargin">
             <number>5</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="phases_layout">
              <property name="leftMargin">
               <number>5</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>5</number>
              </property>
              <property name="bottomMargin">
               <number>5</number>
              </property>
              <item>
               <widget class="QPushButton" name="phase1_button">
                <property name="minimumSize">
                 <size>
                  <width>70</width>
                  <height>35</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="text">
                 <string>Etapa 1</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="phase2_button">
                <property name="minimumSize">
                 <size>
                  <width>70</width>
                  <height>35</height>
                 </size>
                </property>
                <property name="text">
                 <string>Etapa 2</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="phase3_button">
                <property name="minimumSize">
                 <size>
                  <width>70</width>
                  <height>35</height>
                 </size>
                </property>
                <property name="text">
                 <string>Etapa 3</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="calculations_button">
                <property name="minimumSize">
                 <size>
                  <width>70</width>
                  <height>35</height>
                 </size>
                </property>
                <property name="text">
                 <string>Cálculos</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="position_button">
                <property name="minimumSize">
                 <size>
                  <width>70</width>
                  <height>35</height>
                 </size>
                </property>
                <property name="text">
                 <string>Posição</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QStackedWidget" name="phases_stack">
              <property name="currentIndex">
               <number>4</number>
              </property>
              <widget class="QWidget" name="phase1_page">
               <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1,12,2,1">
                <property name="bottomMargin">
                 <number>9</number>
                </property>
                <item>
                 <spacer name="verticalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <layout class="QGridLayout" name="phase1_gridLayout">
                  <property name="leftMargin">
                   <number>30</number>
                  </property>
                  <property name="rightMargin">
                   <number>30</number>
                  </property>
                  <property name="horizontalSpacing">
                   <number>80</number>
                  </property>
                  <property name="verticalSpacing">
                   <number>40</number>
                  </property>
                  <item row="1" column="1">
                   <widget class="QLineEdit" name="lineEdit_4">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="minimumSize">
                     <size>
                      <width>100</width>
                      <height>30</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>500</width>
                      <height>40</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="0">
                   <widget class="QLineEdit" name="lineEdit">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="minimumSize">
                     <size>
                      <width>100</width>
                      <height>30</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>500</width>
                      <height>40</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="1">
                   <widget class="QLineEdit" name="lineEdit_6">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="minimumSize">
                     <size>
                      <width>100</width>
                      <height>30</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>500</width>
                      <height>40</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QLineEdit" name="lineEdit_2">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="minimumSize">
                     <size>
                      <width>100</width>
                      <height>30</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>500</width>
                      <height>40</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="0">
                   <widget class="QLineEdit" name="lineEdit_5">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="minimumSize">
                     <size>
                      <width>100</width>
                      <height>30</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>500</width>
                      <height>40</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="0">
                   <widget class="QLineEdit" name="lineEdit_3">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="minimumSize">
                     <size>
                      <width>100</width>
                      <height>30</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>500</width>
                      <height>40</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <spacer name="verticalSpacer">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout" stretch="11,0,2">
                  <item>
                   <spacer name="horizontalSpacer_2">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item>
                   <widget class="QToolButton" name="continuePhase1_button">
                    <property name="styleSheet">
                     <string notr="true"/>
                    </property>
                    <property name="text">
                     <string>Continuar</string>
                    </property>
                    <property name="icon">
                     <iconset resource="images/resources.qrc">
                      <normaloff>:/Arrow.png</normaloff>:/Arrow.png</iconset>
                    </property>
                    <property name="iconSize">
                     <size>
                      <width>30</width>
                      <height>30</height>
                     </size>
                    </property>
                    <property name="toolButtonStyle">
                     <enum>Qt::ToolButtonTextBesideIcon</enum>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer_3">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="phase2_page">
               <widget class="QToolButton" name="continuePhase2_button">
                <property name="geometry">
                 <rect>
                  <x>360</x>
                  <y>210</y>
                  <width>116</width>
                  <height>37</height>
                 </rect>
                </property>
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="text">
                 <string>Continuar</string>
                </property>
                <property name="icon">
                 <iconset resource="images/resources.qrc">
                  <normaloff>:/Arrow.png</normaloff>:/Arrow.png</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>30</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="toolButtonStyle">
                 <enum>Qt::ToolButtonTextBesideIcon</enum>
                </property>
               </widget>
              </widget>
              <widget class="QWidget" name="phase3_page">
               <widget class="QToolButton" name="continuePhase3_button">
                <property name="geometry">
                 <rect>
                  <x>320</x>
                  <y>190</y>
                  <width>116</width>
                  <height>37</height>
                 </rect>
                </property>
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="text">
                 <string>Continuar</string>
                </property>
                <property name="icon">
                 <iconset resource="images/resources.qrc">
                  <normaloff>:/Arrow.png</normaloff>:/Arrow.png</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>30</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="toolButtonStyle">
                 <enum>Qt::ToolButtonTextBesideIcon</enum>
                </property>
               </widget>
              </widget>
              <widget class="QWidget" name="calculations_page">
               <widget class="QToolButton" name="continueCalculations_button">
                <property name="geometry">
                 <rect>
                  <x>490</x>
                  <y>210</y>
                  <width>116</width>
                  <height>37</height>
                 </rect>
                </property>
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="text">
                 <string>Continuar</string>
                </property>
                <property name="icon">
                 <iconset resource="images/resources.qrc">
                  <normaloff>:/Arrow.png</normaloff>:/Arrow.png</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>30</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="toolButtonStyle">
                 <enum>Qt::ToolButtonTextBesideIcon</enum>
                </property>
               </widget>
              </widget>
              <widget class="QWidget" name="position_page"/>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="after_experiment"/>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="images/resources.qrc"/>
 </resources>
 <connections/>
</ui>
